parsec/backend/postgresql/invite.py

```diff
@@ -588,14 +589,30 @@ async def list(self, organization_id: OrganizationID, greeter: UserID) -> List[I
                     created_on=created_on,
                     status=status,
                 )
-            else:  # Device
+            elif type == InvitationType.DEVICE.str:
                 invitation = DeviceInvitation(
                     greeter_user_id=UserID(greeter),
                     greeter_human_handle=greeter_human_handle,
                     token=token,
                     created_on=created_on,
                     status=status,
                 )
+            elif type == InvitationType.SHAMIR_RECOVERY.str:
+                invitation = ShamirRecoveryInvitation(
+                    greeter_user_id=UserID(greeter),
+                    greeter_human_handle=greeter_human_handle,
+                    token=token,
+                    # TODO: still there ?
+                    claimer_email=claimer_email,
+                    # TODO
```

Comment:
Can you provide more context for the TODO comment ?

Reply:
Need to implement postgres

Reply:
Could you update those TODO comment with the context ?